internal/edgeapi/backend/remote/backend.go

```diff
@@ -123,3 +123,16 @@ func (b *backend) getRegistrationStatus(ctx context.Context, name string, namesp
 	payload := response.Payload
 	return payload, nil
 }
+func (b *backend) GetPlaybookExecutions(ctx context.Context, name string, namespace string) ([]*models.PlaybookExecution, error) {
```

Comment:
Where this method is being used?

Reply:
This PR is in preparation for handling the EdgeConfig and PlaybookExecution CRDs. The next PR will be big, so I'm trying to make it smaller by adding some functions that will be needed. 
This method will be used [here ](https://github.com/project-flotta/flotta-operator/blob/5bfecf1bf926f6ed87fad9afedfd8ee500de26e1/internal/edgeapi/yggdrasil/yggdrasil.go#L167).

Should i remove it?

Reply:
If there are plans of using it, then no need to remove it.
However, it is simpler to evaluate the implementation based on its actual use (for the future)